src/proj04BittingEllmerWang/Controller.java

```diff
@@ -304,8 +377,8 @@ private boolean SaveFile(String content, File file) {
      *              and its source.
      */
     @FXML
-    private void handleExitMenuItem(Event event) {
-        while (tabPane.getSelectionModel().getSelectedItem() != null) {
+    protected void handleExitMenuItem(Event event) {
```

Comment:
why is this protected?

Reply:
Main, within the same package, needs to access it. I'm changing it to default.